README.md

```diff
@@ -1,8 +1,8 @@
-# DB
+# StellarWPDB
 
 [![Tests](https://github.com/stellarwp/db/workflows/Tests/badge.svg)](https://github.com/stellarwp/db/actions?query=branch%3Amain) [![Static Analysis](https://github.com/stellarwp/db/actions/workflows/static-analysis.yml/badge.svg)](https://github.com/stellarwp/db/actions/workflows/static-analysis.yml)
 
-A WPDB wrapper and query builder library.
+A WPDB wrapper and query builder library. Authored by the development team at StellarWP and provided free for the WordPress communinity. Inspired and largely forked from the work in GiveWP. 
```

Comment:
This is a great addition! There's a whole Acknowledgements section at the bottom of this README, but I like the idea of calling out the GiveWP contributions way up top.

```suggestion
A WPDB wrapper and query builder library. Authored by the development team at StellarWP and provided free for the WordPress community.

_Inspired and largely forked from the [GiveWP](https://github.com/impress-org) codebase!_
```